code-challenges.test.js

```diff
@@ -56,4 +120,25 @@ const vowelTester2 = "academy"
 const vowelTester3 = "challenges"
 // Expected output: 2
 
+// going to make another describe statement with an it statement to describe the code and an expect statement to  call the function.
+
+describe("vowelIndex", () => {
+    it("takes in a string and logs the index of the first vowel.", () => {
+        let expectedIndex = [1, 0, 2]
+      expect(vowelIndex(vowelTester1, vowelTester2, vowelTester3)).toEqual(expectedIndex)
+    })
+  })
+
 // b) Create the function that makes the test pass.
+// Now I am going to create a function with conditional if "a, e, i, o, u" then return index of
+
+const vowelIndex = (string) => {
+    if (string == "a, e, i, o, u") {
+      return vowelIndex.indexOf
+    } else if (string !== "a, e, i, o, u") {
+      return "no vowels"
+    }
+  }
+
+  //Again I was not able to get a green test with this one. I feel like my logic is there I just don't know how to execute it with Jest tests.
```

Comment:
For tests, try using a template.  Ex:
`describe("myFunctionName", () => {
  it("describe what the functions intent is", () => {
    expect(myFunctionName(passArgumentsHere)).toEqual(expectedOutput)
  })
})`